internal/plugin/grpc_provider.go

```diff
@@ -176,6 +180,33 @@ func (p *GRPCProvider) GetProviderSchema() providers.GetProviderSchemaResponse {
 	return resp
 }
 
+func (p *GRPCProvider) GetResourceIdentitySchemas() providers.GetResourceIdentitySchemasResponse {
+	var resp providers.GetResourceIdentitySchemasResponse
+
+	resp.IdentityTypes = make(map[string]providers.IdentitySchema)
+
+	protoResp, err := p.client.GetResourceIdentitySchemas(p.ctx, new(proto.GetResourceIdentitySchemas_Request))
```

Comment:
We don't have a capability flag for this one, so we should be checking specifically for an unimplemented method here. I can't remember the best way to do this, but I thin the grpc framework will return `codes.Unimplemented`